.drone/drone.yml

```diff
@@ -54,48 +54,6 @@ steps:
 depends_on:
   - Lint
 
----
```

Comment:
You'll also need to look for other pipelines that are referencing E2E and remove the reference :)